### PR TITLE
tooltip and popover

### PR DESCRIPTION
**Pull request by ramitos**
image
image

### CODE REVIEW

File: packages/ui-toolkit/src/boxes/index.js [hunk 1]
```diff
@@ -11,19 +11,9 @@ export const tooltipShadow = `0 ${remcalc(2)} ${remcalc(6)} ${remcalc(
...
-  confirmed: css`
-    ${remcalc(1)} solid ${props => props.theme.grey};
-  `,
-  error: css`
-    ${remcalc(1)} solid ${props => props.theme.red};
-  `,
-  secondary: css`
-    ${remcalc(1)} solid ${props => props.theme.secondaryActive};
-  `
+  checked: css`${remcalc(1)} solid ${props => props.theme.primary};`,
```
Comment (SaraVieira) [inline packages/ui-toolkit/src/boxes/index.js]: prettier :(

File: packages/ui-toolkit/src/styleguide/wrapper.js [hunk 2]
```diff
@@ -1,59 +1,21 @@
...
 
 const StyledBase = Base.extend`
   /* trick prettier */
   background-color: transparent;
 `;
 
-export default class Wrapper extends Component {
-  componentWillMount() {
-    // eslint-disable-next-line no-unused-expressions
-    injectGlobal`
```
Comment (SaraVieira) [inline packages/ui-toolkit/src/styleguide/wrapper.js]: We dont need this anynore ?

File: packages/ui-toolkit/src/tooltip/container.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    }
+
+    this.outsideTap = outy(
+      elements,
+      ['click', 'touchstart'],
+      this._handleOutsideTap
+    );
+  };
+
+  _handleOutsideTap = ev =>
```
Comment (SaraVieira) [inline packages/ui-toolkit/src/tooltip/container.js]: do you like this _ convention ?

Comment (ramitos) [reply packages/ui-toolkit/src/tooltip/container.js]: yeah, for "private" methods.

File: packages/ui-toolkit/src/styleguide/wrapper.js (hunk 2 above)
Comment (ramitos) [reply packages/ui-toolkit/src/styleguide/wrapper.js]: Yeah, we can just use `<RootContainer />` ;)

Comment (SaraVieira) [reply packages/ui-toolkit/src/styleguide/wrapper.js]: Awesome !

File: packages/ui-toolkit/src/tooltip/container.js (hunk 3 above)
Comment (SaraVieira) [reply packages/ui-toolkit/src/tooltip/container.js]: Carry one then sire :)

File: packages/ui-toolkit/src/boxes/index.js (hunk 1 above)
Comment (ramitos) [reply packages/ui-toolkit/src/boxes/index.js]: I think the unreleased version of prettier handles that better

**Pull request closed without merging**